low-level/solution/lr2/server/health.py

```diff
@@ -161,12 +161,12 @@ def set_health_data(health_data: dict, status, description=None,
                         recommendation=None, specifics=None):
         """Sets health attributes for a component."""
         good_state = (status == "OK")
-        if not description:
+        if description in const.RESOURCE_MAP["undesired_values"]:
             description = "%s %s in good health." % (
                 health_data.get("uid"),
                 'is' if good_state else 'is not')
-        if not recommendation:
-            recommendation = 'NA' if good_state\
+        if recommendation in const.RESOURCE_MAP["undesired_values"]:
+            recommendation = 'NOT AVAILABLE' if good_state\
```

Comment:
if in good state, recommendation needs to be None !

Reply:
Done.

Reply:
This gives below result, @cdeshmukh 
recommendation value will be shown as null.
```
            {
              "uid": "eth3",
              "fru": "false",
              "last_updated": 1628057623,
              "health": {
                "status": "OK",
                "description": "Network Interface 'eth3' is in good health.",
                "recommendation": null,
                "specifics": [
                  {
                    "networkErrors": 0,
                    "droppedPacketsIn": 0,
                    "droppedPacketsOut": 0,
                    "packetsIn": 13413208,
                    "packetsOut": 12970,
                    "trafficIn": 1069059791,
                    "trafficOut": 1114812,
                    "nwStatus": "UP",
                    "nwCableConnStatus": "CONNECTED"
                  }
                ]
              }
```

Reply:
>   "recommendation": null,

Need None as value, wherever health is seen good.

Reply:
Done.